armi/operators/operator.py

```diff
@@ -387,17 +387,26 @@ def _performTightCoupling(self, cycle: int, timeNode: int, writeDB: bool = True)
         -----
         writeDB is False for OperatorSnapshots as the DB gets written at EOL.
         """
-        if self.couplingIsActive():
+        if not self.couplingIsActive():
+            # no coupling was requested
+            return
+        skipCycles = (int(val) for val in self.cs["cyclesSkipTightCouplingInteraction"])
+        if cycle in skipCycles:
+            runLog.warning(
+                f"interactAllCoupled disabled this cycle ({self.r.p.cycle}) due to "
+                "`cycleTreatTightConverged` setting."
```

Comment:
Typo here. the setting name in the runLog.warning doesn't match the actual setting name.

Reply:
I'll address this in https://github.com/terrapower/armi/pull/1172. No action needed.